### PR TITLE
Added new FAQs and context in the docs based on feedback in matrix

### PR DESCRIPTION
**Pull request by scottharwell**
* Updated FAQ Page
* Added footer references to related FAQ questions
* Added new sections to installing extension and logging in
* Added `pre-commit` with markdownlint

### CODE REVIEW

File: docs/introduction.md [hunk 1]
```diff
@@ -11,21 +11,23 @@ Ansible Lightspeed with Watson Code Assistant intends to be your expert referenc
...
+The Ansible VS Code extension, with Ansible Lightspeed with Watson Code Assistant features enabled, automatically collects recommendations, usage telemetry, and playbook state through automated events. Telemetry regarding how Ansible Lightspeed with Watson Code Assistant is used is an important data that allows Red Hat to improve the service over time. The following table identifies all of the data collected from the VS Code extension and returned to Ansible Lightspeed with Watson Code Assistant.
```
Comment (robinbobbitt) [inline docs/introduction.md:14]: nit: "is an important data" needs to be fixed

**Review by robinbobbitt — CHANGES_REQUESTED** [review]

File: docs/vscode_guide/installing_vs.md [hunk 2]
```diff
@@ -43,14 +43,24 @@ To configure the Ansible VS Code extension for Ansible Lightspeed with Watson Co
...
+## Logging In to Ansible Lightspeed
+
+1. Open the **Ansible** extension icon on the left-hand side of the Visual Studio Code interface and click **Connect**.
+2. Follow the prompts to sign in to Ansible Lightspeed with Watson Code Assistant using your GitHub credentials.[^2]
+3. Carefully review the Terms and **Agree** or **Disagree**. If you click Disagree, you will not be able to complete the login. Lastly, follow the prompt to allow VSCode to interact with Ansible Lightspeed with Watson Code Assistant on your behalf.
+4. Upon successful completion of the login flow, you should see a “Logged in as \<your GitHub id\>” message in the Ansible side bar.
+
+## Logging Out of Ansible Lightspeed
+
+To logout of Ansible Lightspeed, click on the "person" icon in the bottom-left panel of the VS Code window.  This icon will have a list of accounts that VS Code is logged into.  Click "Ansible Lightspeed", then "Logout" to logout of the service.
```
[the remark this answers is not in the document]
Comment (scottharwell) [reply docs/vscode_guide/installing_vs.md]: Good call out.  Added.

File: docs/introduction.md [hunk 3]
```diff
@@ -11,21 +11,23 @@ Ansible Lightspeed with Watson Code Assistant intends to be your expert referenc
...
+The Ansible VS Code extension, with Ansible Lightspeed with Watson Code Assistant features enabled, automatically collects recommendations, usage telemetry, and playbook state through automated events. Ansible Lightspeed with Watson Code Assistant telemetry is used allows Red Hat to improve the service over time. The following table identifies all of the data collected from the VS Code extension and returned to Ansible Lightspeed with Watson Code Assistant.
```
Comment (robinbobbitt) [inline docs/introduction.md:14]: One more small edit: is used allows

File: docs/vscode_guide/installing_vs.md [hunk 4]
```diff
@@ -43,14 +43,32 @@ To configure the Ansible VS Code extension for Ansible Lightspeed with Watson Co
...
+
+## Logging Out of Ansible Lightspeed
+
+To logout of Ansible Lightspeed, follow these steps:
+
+1. Click on the "person" icon in the bottom-left panel of the VS Code window.  This icon will have a list of accounts that VS Code is logged into.  
+2. Click "Ansible Lightspeed", then "Logout" to logout of the service.
+3. Open your browser.
+4. Navigate to [https://c.ai.ansible.redhat.com/][base-path].
+5. Click "Logout".
```
Comment (robinbobbitt) [inline docs/vscode_guide/installing_vs.md:64]: The button is actually two words: "Log out".

**Review by robinbobbitt — CHANGES_REQUESTED** [review]
Looks great but needs two more super small edits.

File: docs/vscode_guide/installing_vs.md (hunk 4 above)
Comment (scottharwell) [reply docs/vscode_guide/installing_vs.md]: Corrected.  And, VS Code uses "Sign Out" while the web page uses "Log Out"

**Review by robinbobbitt — APPROVED** [review]

**Pull request merged**